app/assets/stylesheets/z_custom.scss

```diff
@@ -0,0 +1,302 @@
+/* COLUMNS */
+
+.column, .columns{
+  padding: 20px 0px;
+}
+.container.staticpages {
+  max-width: 100%;
+}
+
+.inner-container{
+  max-width: 1200px;
+  margin: 0 auto;
+}
+/* FONTS */
+h1, h2, h3, h4, h5, h6, .main-nav{
+  font-family: 'Oswald', sans-serif;
+  color:#fff;
+  text-transform:uppercase;
+  letter-spacing:.006em;
+  text-shadow: 1px 1px 2px rgba(0, 0, 0, 1);
+}
+h1, h2, h6{
+  font-family:'Arvo', serif;
+  font-weight:bold;
+  letter-spacing:.006em;
+}
+h6{
+  text-align:center;
+  min-height:50px;
+}
+
+h5{
+  text-align:center;
+}
+a{
+  text-decoration:none;
+  color:#fff;
+}
+
+a:hover{
+  color:#8a1f1a;
+}
+
+p, table{
+  color:#fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space